test/e2e/framework/create.go

```diff
@@ -287,18 +288,27 @@ var factories = map[What]ItemFactory{
 	{"StorageClass"}:       &storageClassFactory{},
 }
 
-// PatchName makes the name of some item unique by appending the
-// generated unique name.
-func (f *Framework) PatchName(item *string) {
+// uniquifyName makes the name of some item unique per namespace by appending the
+// generated unique name of the test namespace.
+func (f *Framework) uniquifyName(item *string) {
```

Comment:
This is still an API change. When this create.go was initially merged, I was explicitly asked to split it up so that someone can load items, patch them (either with the existing `PatchItems` or some custom code), and then create them.

Such custom code is meant to have access to individual building blocks like this `PatchName`.

Reply:
feel free to submit a PR changing these back to external